### PR TITLE
SJSU-1035_3, History Page

### PR DESCRIPTION
**Pull request by SandroSal**
new styling for History page.

what I changed:
- cancellationPrompt.js : just modified the dates, since I changed the format they are passed
- HistoryExpansionTable.js : broke this part out of the main component
- HistoryOverview.js : added rewards points overview, most other changes is just shifting code around
- history.css : added min-height so it won't display the footer in the middle of the page
- profileReducer: took out the dummy values

### CODE REVIEW

File: front_end/src/components/history/history.css [hunk 1]
```diff
@@ -235,3 +253,12 @@
...
 }
+
+.rewardsPointsBalance {
+  color: #3d4e96;
+  text-align: right;
+  font-size: 20px;
+  font-weight: 600;
+  margin-bottom: 0px;
+  margin-right: 25px;
+}
```
Comment (DaleCS) [inline front_end/src/components/history/history.css]: Couldn't highlight it but there is a class named .HistoryPageDates and another called .HistoryContainerExpand both aren't being used.

File: front_end/src/components/history/HistoryOverview.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
+// TODO:
+// - IF CHANGE, SHOW OLD/NEW DATES
+
 import React, { Component } from "react";
 import { withStyles, withTheme, withWidth } from "@material-ui/core/styles";
```
Comment (DaleCS) [inline front_end/src/components/history/HistoryOverview.js]: withTheme and withWidth were imported but never used.

Comment (SandroSal) [reply front_end/src/components/history/HistoryOverview.js]: removed

**Comment by SandroSal** [comment]
- added code so the panel won't expand on change-button-click
- buttons are part of other stories. They will be designed by Yu and Minh respectively.
- removed the unused imports

**Pull request merged**